lineapy/plugins/mlflow_io.py

```diff
@@ -0,0 +1,53 @@
+"""
```

Comment:
Right now plugins is all about writing Pipelines, I wonder if we should break the directory down even further because this is not used for writing pipelines at all. 

Something like create `lineapy/plugins/serializers` and move all current `lineapy/plugins/*` to  `lineapy/plugins/pipeline`. Probably not necessary for this PR but would be much clearer in my opinion.

Reply:
Agree here, I move new stuff into `lineapy/plugins/serializers` and keep existing stuff for pipeline.